core/src/main/java/hudson/scheduler/CronTabList.java

```diff
@@ -131,7 +131,7 @@ public static CronTabList create(@Nonnull String format, Hash hash) throws ANTLR
     public @CheckForNull Calendar previous() {
         Calendar nearest = null;
         for (CronTab tab : tabs) {
-            Calendar scheduled = tab.floor(Calendar.getInstance());
+            Calendar scheduled = tab.floor(tab.getTimeZone() == null ? Calendar.getInstance() : Calendar.getInstance(tab.getTimeZone()));
```

Comment:
FindBugs will likely gruble about that. Better to use a temporary variable for `tab.getTimeZone()`